pkg/binfmt/Makefile

```diff
@@ -13,3 +13,11 @@ tag: $(DEPS)
 push: tag
 	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
 	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+
+check:
+	@if curl -sSL https://registry.hub.docker.com/v2/repositories/$(ORG)/$(IMAGE)/tags/$(HASH) | grep -q 'Not found'; then\
```

Comment:
you should not use `curl` for this. Especially as it ties it to docker hub. whats wrong with using `pull` to see if it exists? Especially as we want to test it exists and is signed anyway